scripts/package/README.md

```diff
@@ -90,31 +102,41 @@ This script cannot download the sdk's by itself, but instead relies on the user
 
 Run the script
 
-	./scripts/package/package_xcode_and_sdks.sh
+```sh
+	$ ./scripts/package/package_xcode_and_sdks.sh
+```
 
 and it will output files in `local_sdks` (version depends on which is the current recommended version):
 
+```
 	./local_sdks/MacOSX15.2.sdk.tar.gz
 	./local_sdks/XcodeDefault16.2.xctoolchain.darwin.tar.gz
 	./local_sdks/iPhoneOS18.2.sdk.tar.gz
 	./local_sdks/iPhoneSimulator18.2.sdk.tar.gz
+```
 
 ## Windows
 
 Run the script
 
-	./scripts/package/package_win32_sdk.sh
+```sh
+	$ ./scripts/package/package_win32_sdk.sh
+```
 
 ## HTML5
 
 The installation of the HTML5 compiler is done by calling the build function `install_ems`:
 
+```sh
 	$ ./scripts/build.py install_ems
+```
 
 ## Linux
 
-We use packages from [Clang](https://github.com/llvm/llvm-project/releases) for the Ubuntu16.04 target.
-Current version: https://github.com/llvm/llvm-project/releases/download/llvmorg-13.0.0/clang+llvm-13.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz
+We use packages from [Clang](https://github.com/llvm/llvm-project/releases) for the Ubuntu18.04 target.
+Current version:
+* (x86_64) https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.0/clang+llvm-16.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz
+* (arm64) https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.0/clang+llvm-16.0.0-aarch64-linux-gnu.tar.xz
```

Comment:
Note that we've just updated our ci.py to install clang 17, just as the Extender uses.
Theoretically we should update these packages as well.

However, using the llvm.sh script to install clang is so easy to do, and I see little use for us anymore to packages the linux clang versions.